platform/platform-app/src/main/resources/features.xml

```diff
@@ -167,9 +177,11 @@
         <feature version="${camel.version}">camel-core</feature>
         <feature version="${spring-version-range}">spring</feature>
         <feature version="${spring-version-range}">spring-jms</feature>
-        <bundle dependency="true">mvn:org.apache.geronimo.specs/geronimo-jta_1.1_spec/${geronimo-jta-spec-version}</bundle>
+        <!-- Use version 1.2 of javax transaction, DDF update to the camel feature file -->
+        <feature>transaction</feature>
+        <!-- Use version 2.0 of javax jms, DDF update to the camel feature file -->
+        <bundle dependency="true">mvn:org.apache.geronimo.specs/geronimo-jms_2.0_spec/${geronimo-jms_2_spec.version}</bundle>
```

Comment:
This property should be `geronimo-jms2-spec-version` instead.

We're trying to keep everything named the same as the original file that these features were copied from https://github.com/apache/camel/blob/camel-2.19.0/pom.xml.

And move `<geronimo-jms_2_spec.version>1.0-alpha-2</geronimo-jms_2_spec.version>` property in the root pom to [this section of the root pom](https://github.com/codice/ddf/blob/ac97624deeb4e454247631a5c1211df51e6d37ab/pom.xml#L121-L129).

And modify the `<geronimo-jms2-spec-version>1.0-alpha-2</geronimo-jms2-spec-version>` in the platform-app pom to be `<geronimo-jms2-spec-version>${geronimo-jms_2_spec.version}</geronimo-jms2-spec-version>`, like the other properties in [that section of the root pom](https://github.com/codice/ddf/blob/ac97624deeb4e454247631a5c1211df51e6d37ab/pom.xml#L121-L1290).